api/views/search.py

```diff
@@ -0,0 +1,30 @@
+from api import app, db
+from api.models import Story, StoryPOI, POI
+from api.utils import create_response, row_constructor
+from flask import Blueprint, request
+
+SEARCH_URL = "/search"
+
+mod = Blueprint('search', __name__)
+
+@app.route(SEARCH_URL, methods=['GET'])
+def search_pois():
+    data = request.args
+    pois = []
+    pois1 = []
+    pois2 = []
+
+    if data.get('q') is not None:
+        if data.get('name') is not None:
+            if data.get('name') == "True":
```

Comment:
be careful of using the string `"True"` because as a string, it depends on how it's sent from the request. i don't think there's necessarily a standard for sending booleans as query params, but keep in mind that the stringified value of `True` in javascript is `"true"` not `"True"`